--- config.php-dist	2011-11-10 21:25:19.909372305 +0400
+++ config.php-dist.mysql	2011-11-10 23:33:33.049370586 +0400
@@ -1,10 +1,12 @@
 <?php
 	// Database server configuration
-	define('DB_TYPE', "pgsql"); // or mysql
-	define('DB_HOST', "localhost");
-	define('DB_USER', "fox");
-	define('DB_NAME', "fox");
-	define('DB_PASS', "XXXXXX");
+	// dbconfig-common
+	include_once('/etc/tt-rss-mysql/database.php');
+	define('DB_TYPE', $dbtype);
+	define('DB_HOST', $dbserver);
+	define('DB_USER', $dbuser);
+	define('DB_NAME', $dbname);
+	define('DB_PASS', $dbpass);
 	//define('DB_PORT', '5432'); // when neeeded, PG-only
 
 	define('SELF_URL_PATH', 'http://yourserver/tt-rss/');
@@ -17,7 +19,7 @@
 	// Operate in single user mode, disables all functionality related to
 	// multiple users.
 	
-	define('CACHE_DIR', 'cache');
+	define('CACHE_DIR', '/var/cache/tt-rss');
 	// Local cache directory for RSS feed content.
 
 	define('SIMPLEPIE_CACHE_IMAGES', false);
@@ -130,7 +132,7 @@
 	// You want new users to be automaticaly created in tt-rss database
 	// on first login
 
-	define('LOCK_DIRECTORY', 'lock');
+	define('LOCK_DIRECTORY', '/var/lock/tt-rss');
 	// Directory for lockfiles, must be writable to the user you run
 	// daemon process or cronjobs under.
 
